chef-provisioning-fog.gemspec

```diff
@@ -12,12 +12,18 @@ Gem::Specification.new do |s|
   s.email = ['jkeiser@getchef.com', 'hh@vulk.co', 't@vulk.co', 'w@vulk.co']
   s.homepage = 'https://github.com/opscode/chef-provisioning-fog'
 
+  s.add_dependnecy 'chef'
```

Comment:
`chef-provisioning` should pull this in automatically - can you remove this and run `bundle update` to check if chef is still being pulled in?